libraries/types/pbft_block/include/pbft/pbft_block.hpp

```diff
@@ -15,6 +15,57 @@ namespace taraxa {
  * @{
  */
 
+class PbftBlockExtraData {
```

Comment:
I think it's better to move it to separate files and have this version related things in a separate structure, so it will be easier to extend it later

Reply:
yes and pls also separate implementations into the cpp file

Reply:
Done